### DISCUSSION TITLE
Globoplay Videos

### DISCUSSION THREAD

**Discussion by crazypython-rgb** [aclap-dev/video-downloadhelper#1582] [Help and Q&A]
### Your question

Some videos of recent series in Globoplay Brazil get downloaded all green with corrupted/ encripted video data. Any way to de-scramble them?

### Addon & CoApp version and details

Some videos of recent series in Globoplay Brazil get downloaded all green with corrupted/ encripted video data. Any way to de-scramble them?

**Comment by Mattmjs** [comment]
Probably DRM used on the site in question and unfortunately the file is useless. No way to decrypt them.
You'd have to screen record it. I'll leave that for you to research on.
